app/frontend/stores/permit-application-store.ts

```diff
@@ -52,15 +53,21 @@ export const PermitApplicationStoreModel = types
         jurisdiction: pad.jurisdiction.id,
         submitter: pad.submitter.id,
         stepCode: pad.stepCode?.id,
+        isFullyLoaded: true,
       })
     },
     __beforeMergeUpdateAll(permitApplicationsData) {
+      // return the unaltered data if there are no jurisdictions, indicating that this is the base view
+      if (!permitApplicationsData.some((pad) => pad.jurisdiction && pad.submitter && pad.stepCode))
+        return permitApplicationsData
+
```

Comment:
Just clarifying if all permit applications are expected to have a stepCode? Because if the jurisdiction and submitter is present but stepCode is missing this will still early return

Reply:
I think I have to change these to ORs